dawn/src/dawn/IIR/Stage.cpp

```diff
@@ -280,47 +282,63 @@ void Stage::appendDoMethod(DoMethodSmartPtr_t& from, DoMethodSmartPtr_t& to,
                            std::make_move_iterator(from->getAST().getStatements().end()));
 }
 
-std::vector<std::unique_ptr<Stage>> Stage::split(std::deque<int>& splitterIndices,
-                                                 std::deque<DependencyGraphAccesses>* graphs) {
+namespace {
```

Comment:
Our style guide seems to mandate static, it would have been helpful if you would have just said that, because now I had to search for it...
My first response would have been, why static function over unnamed namespace. ;)

Reply:
done